src/hooks/useSynth.tsx

```diff
@@ -23,17 +23,26 @@ export const useSynth = () => {
     setFreq(value);
   };
 
-  const newWave = (newType: ToneOscillatorType): void => {
+  const newWave = (newType: Tone.ToneOscillatorType): void => {
     selectedTrack.synth.type = newType;
   };
 
+  const rhythm = 8;
   const startSynth = (): void => {
+    const osc = selectedTrack.synth;
     setStarted(true);
-    selectedTrack.synth.start();
+    Tone.Transport.start();
+    // repeated event every 8th note
+    Tone.Transport.scheduleRepeat((time) => {
+      // use the callback time to schedule events
+      osc.start(time).stop(time + 0.1);
+    }, `${rhythm}n`);
+    // selectedTrack.synth.start(); commented out just for now, to test out sequencer functionality
   };
   const stopSynth = (): void => {
     setStarted(false);
-    selectedTrack.synth.stop();
+    Tone.Transport.stop();
+    // selectedTrack.synth.stop(); todo: trying to figure out how to connect these events
```

Comment:
So does this stop all synth instances or is that what your todo is referring to?

Reply:
I believe it currently stops `selectedTrack`. I'm not sure what that to-do means, it might have been when I was still trying to figure out how to run the sequence through the oscillator. Now I'm wondering if I forgot to push that commit. Will check later